frontend/src/routes/sudo/health/+page.svelte

```diff
@@ -51,8 +51,7 @@
 					'All active IPs have correct PTR records (Forward and Reverse)',
 				instance_dkim_correct: 'Instance DKIM is correct',
 				all_ips_are_in_spf_record: 'All IPs are in SPF record',
-				all_servers_can_be_reached_via_private_network:
-					'All servers can be reached via private network'
+				no_unread_infrastructure_bounces: 'No unread infrastructure bounces'
```

Comment:
I just noticed this function and HealthCheckItem's formatCheckName is the same. Abstract it away to prevent duplications